micrometer-observation-test/src/test/java/io/micrometer/observation/tck/TestObservationRegistryAssertTests.java

```diff
@@ -64,7 +64,7 @@ void should_not_break_on_multiple_threads() {
 
     @Test
     void should_fail_when_observation_not_started() {
-        Observation.createNotStarted("foo", registry).stop();
+        Observation.createNotStarted("foo", registry);
```

Comment:
I needed to "fix" this test: it simulates an invalid scenario but it does not need to call stop on an `Observation` that hasn't been started to verify the expected behavior.